lib/Common/Core/EtwTraceCore.h

```diff
@@ -47,6 +47,10 @@
 #define JS_ETW(s) s
 #define IS_JS_ETW(s) s
 
+#ifdef ENABLE_JS_LTTNG
+#include "jscriptEtw.h"
```

Comment:
this is somewhat confusing here (if lttng { include etw }) -- can the output file be `jscriptLTTng.h` instead? (Or, even better, `JScriptLTTng.h` or `JScriptLttng.h`

Reply:
The `jscriptEtw.h` file is kind of the cross-over point between ETW and our LTTng support: it provides an ETW-like interface (e.g. `EmitEvent*` functions) that call into our LTTng functions. If you still think it is confusing, I don't mind changing the name.